Documentation~/en/videostreaming.md

```diff
@@ -69,7 +69,7 @@ foreach(var track in listTrack)
 }
 ```
 
-ハードウェアエンコーダーを選択している場合、グラフィックデバイスの制約によって、同時に利用可能なトラック数が制限される場合があります。一般的に NVIDIA Geforce で同時に利用可能なビデオトラック数は **2本** までです。詳しくは [NVDIA Codec SDK のドキュメント](https://developer.nvidia.com/video-encode-decode-gpu-support-matrix) を参照してください。
+When using hardware encoding, the number of tracks that can be used simultaneously may be limited depending on the graphic device's limitations. Generally, up to **two tracks** can be used simultaneously on an NVIDIA Geforce card. For details, see the [NVDIA Codec SDK documentation](https://developer.nvidia.com/video-encode-decode-gpu-support-matrix).

```

Comment:
I suggest making this section clearer. Many developers are probably going to run this code on servers, and possibly in the cloud. The example of 4 tracks is for Nvidia Titan M60, used on [Microsoft Azure NV-series virtual machines](https://docs.microsoft.com/en-us/azure/virtual-machines/sizes-gpu).

```suggestion
When using hardware encoding, the number of tracks that can be used simultaneously may be limited depending on the graphic device's limitations. Generally, on desktop GPUs, up to **two tracks** can be used simultaneously on an NVIDIA Geforce card. (On server-grade GPUs this is typically 4.) For details, see the [NVDIA Codec SDK documentation](https://developer.nvidia.com/video-encode-decode-gpu-support-matrix).

```